source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,362 @@
+the backend. If your object models don't have a partition key field, the
+backend automatically injects one into every object the Partition-Based Sync
+client creates.
+
+These are some other characteristics of the injected field you should be aware
+of:
+
+- The injected field follows this structure: ``<yourPartitionKey> == 
+  <yourPartitionValue>``. The field key is derived from your old
+  Partition-Based Sync  **backend**  configuration. The value is derived from
+  your current Partition-Based Sync  **client**  configuration.
+- The injected field does not sync back to the client that created the object.
+  This means the originating client won't be able to read the injected
+  ``partitionKey`` field.
+
+The Partition-Based Sync client automatically creates sync subscriptions for 
+each table where ``partitionKey == partitionValue``. This will continue
+until you :ref:`migrate your client app <realm-sync-migrate-client>` to
+Flexible Sync.
+
+After migrating your client app to Flexible Sync, the backend will stop injecting
+the partition key field. This means objects created after migrating your client
+app to Flexible Sync may not be available in clients that haven't yet been
+updated to Flexible Sync.
+
+.. _realm-sync-migrate-client:
+
+Migrate Client App to Flexible Sync
+-----------------------------------
+
+After your app's backend has been migrated to Flexible Sync, we recommend
+that you migrate your client app to Flexible Sync. When you migrate your
+client app to Flexible Sync, the client will stop automatically creating
+sync subscriptions for each Realm object model in your app's data model.
+
+We also recommend removing all sync subscriptions and then creating new subscriptions
+for your data. This is the clearest way to control what data is synced after a
+migration. If you want to remove individual subscriptions, the automatically
+generated subscriptions use this naming format: ``flx_migrated_<objecModelName>``.
+For example, a subscription for the ``Person`` collection would be named
+``flx_migrated_Person``.
+
+Refer to the SDK documentation for details about adding and removing
+subscriptions:
+
+- Java SDK
+   :ref:`Flexible Sync - Java SDK <java-flexible-sync>`
+
+- Kotlin SDK
+   :ref:`Subscribe to Queryable Fields - Kotlin SDK <kotlin-subscriptions>`
+
+- .NET SDK
+   :ref:`Manage Flexible Sync Subscriptions - .NET SDK <dotnet-flexible-sync>`
+
+- Node.js SDK
+   :ref:`Flexible Sync - Node.js SDK <node-flexible-sync>`
+
+- React Native SDK
+   :ref:`Flexible Sync - React Native SDK <react-native-flexible-sync>`
+
+- Swift SDK
+   :ref:`Manage Flexible Sync Subscriptions - Swift SDK <swift-manage-flexible-sync-subscriptions>`
+
+You can keep the automatically-generated subscriptions if you don't want to
+remove and recreate them. But you should consider that because these 
+subscriptions aren't created in your client code, they may make it harder to
+maintain and extend client code.
```

Comment:
And they will not exist on any clients that never connected to the App after the migration to Flexible Sync was performed.